src/components/application_manager/src/commands/mobile/add_sub_menu_request.cc

```diff
@@ -59,7 +59,7 @@ void AddSubMenuRequest::Run() {
 
   const int32_t menu_id =
       (*message_)[strings::msg_params][strings::menu_id].asInt();
-  if (app->FindSubMenu(menu_id)) {
+  if (!app->FindSubMenu(menu_id)) {
```

Comment:
@AByzhynar, removed in 711c3da because of initial misunderstanding of the meaning of this expression.